modules/core/src/main/java/org/apache/fluo/core/worker/NotificationProcessor.java

```diff
@@ -77,8 +77,8 @@ public Integer getValue() {
 
     private long size(RowColumn rowCol) {
       Column col = rowCol.getColumn();
-      return rowCol.getRow().length() + col.getFamily().length() + col.getQualifier().length()
-          + col.getVisibility().length();
+      return (long) rowCol.getRow().length() + col.getFamily().length()
```

Comment:
Will this make the expression be evaluated as a long?  Was it being evaluated as an int before and then converted to long at the end?

Reply:
@keith-turner Exactly! Expression is evaluated as int and than converted to long which may conclude with an undesired result.